tests/test_dataset.py

```diff
@@ -367,7 +367,10 @@ def test_fill_in_defaults():
 
     ds.fill_in_defaults(dataset_from_repo)
     assert ds.format == "parquet"  # NB: not csv
-    assert ds.compression is None
+    if ds.format == "parquet":
+        assert ds.compression == "snappy"
+    else:
+        assert ds.compression is None
```

Comment:
```suggestion
    assert ds.compression == "snappy"
```

This is the only path that would ever be taken, no?